db/mongo/mongoController.js

```diff
@@ -0,0 +1,33 @@
+require('dotenv').config();
+const { MongoClient } = require('mongodb');
+
+const {
+  MONGO_ADDRESS,
+  MONGO_DB_NAME,
+  MONGO_COLLECTION,
+} = process.env;
+
+let client = null;
+let collection = null;
```

Comment:
I don't totally understand the reason for declaring client and collection as null before you make the connection, is it in the case of the MongoClient.connect() failing?

Reply:
yeah, i guess it's not really that different from leaving them undefined. I think you're right, I'll update that to a more standard variable declaration!